lib/linguist/heuristics.yml

```diff
@@ -232,6 +247,7 @@ disambiguations:
     - '^\s*(?:public\s+|private\s+|\s*)function\s+\w+\s*\([\w\s,]*\)\s*{'
   - language: POV-Ray SDL
     pattern: '^\s*#(declare|local|macro|while)\s'
+  - language: C++
```

Comment:
Please remove this. It's out-of-scope for this PR, and will also misclassify files for the [reason I described above](#yup-this-reason).